modules/darwin/common/default.nix

```
@@ -66,5 +66,11 @@
     StrictModes no
   '';
 
+  # Make sure to disable netbios on activation
+  system.activationScripts.no-netbios.text = ''
```

`nix-darwin` doesn't have support for arbitrary activationScripts, need to use pre/post.
```suggestion
  system.activationScripts.postActivation.text = ''
```

---

Good to know, thanks